_adapter.ts

```diff
@@ -47,7 +47,6 @@ export interface HttpConn extends AsyncIterable<RequestEvent> {
 }
 
 export function nativeAdapter(conn: Deno.Conn): HttpApiAdapter {
-  // @ts-ignore
```

Comment:
Keep this. Running without `--unstable` flag will cause compile error.

Reply:
Sorry, Fixed it.